examples/energy-management-app/energy-management-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -33,6 +34,31 @@ using chip::Protocols::InteractionModel::Status;
  */
 CHIP_ERROR GetEpochTS(uint32_t & chipEpoch);
 
+/**
+ * @brief   Helper function to get current timestamp and work out the day of week based on localtime
+ *
+ * @param   reference to hold the day of week as a bitmap
+ *
+ * Sunday = 0x01, Monday = 0x01 ... Saturday = 0x40 (1<<6)
```

Comment:
are there constants for these? Why are we not using a BitMask type with a strongly typed enum in it? That one has RawValue if we need the actual raw bits.

Reply:
I have changed the comment. This bitmap is effectively that defined by chip::app::Clusters::EnergyEvse::TargetDayOfWeekBitmap. We map from a Unix day (Sunday = 0, Monday =1 ... ) to the equivalent bitmask.
I guess ideally we could have returned a chip::BitMask instead of a uint8_t. However would be acceptable to leave as a uint8_t as it ripples through a couple of layers of functions please?

Reply:
GetEpochTS(), GetLocalDayOfWeekFromUnixEpoch(), GetLocalDayOfWeekNow(), GetMinutesPastMidnight() have now been moved into a separate module (examples/energy-management-app/energy-management-common/include/EnergyTimeUtils.h and examples/energy-management-app/energy-management-common/src/EnergyTimeUtils.cpp). The uint8_t bitmask has also been replaced with BitMask<EnergyEvse::TargetDayOfWeekBitmap>